neural_lam/train_model.py

```diff
@@ -300,36 +308,14 @@ def main(input_args=None):
     # Only init once, on rank 0 only
     if trainer.global_rank == 0:
         utils.init_wandb_metrics(
-            logger, args.val_steps_to_log
+            logger, val_steps=args.val_steps_to_log
         )  # Do after wandb.init
-
+        # TODO: should we save the datastore config here?
+        # wandb.save()
```

Comment:
Yes, that would be useful I think.

Reply:
I added this in https://github.com/leifdenby/neural-lam/commit/a955ceecf1a7b49b81a2b4f381cafde9727733eb - I don't know if this will work with other loggers though, but it works in wandb:
![Screenshot 2024-10-01 at 15 55 23](https://github.com/user-attachments/assets/2aa33ec7-29e8-4439-ae27-dfd80ffd4499)